app/ui/src/app/core/providers/form-factory-provider.service.ts

```
@@ -217,11 +217,13 @@ export class FormFactoryProviderService extends FormFactoryService {
                          field: ConfiguredConfigurationProperty,
                          value: any
   ) {
+    // bandaid for rest service returning string 'true' instead of boolean true
+    let defaultValue = value || field.value || field.defaultValue;
```

`defaultValue` is probably a misleading name here?